src/dev/run_i18n_extract.ts

```diff
@@ -52,13 +46,17 @@ run(
         `${chalk.white.bgRed(' I18N ERROR ')} --path and --include-config require a value`
       );
     }
-
-    const config = await mergeConfigs(includeConfig);
+    const srcPaths = Array().concat(path || ['./src', './packages', './x-pack']);
```

Comment:
This can be:

```js
const srcPaths = path ? [path] : ['./src', './packages', './x-pack'];
```

Reply:
`path` can be either an array or a string thats why i've used `concat` here and avoided 3 conditionals